examples/data/preprocessing/earthquake_data.py

```diff
@@ -24,23 +26,18 @@
         df = pd.read_csv('%d_%d.csv' % (i, m), dtype={'nst': 'float64'})
         dfs.append(df)
 df = pd.concat(dfs, sort=True)
-df.to_parquet('../earthquakes.parq', 'fastparquet')
+df.to_parquet(os.path.join(THIS_DIR, '..', 'earthquakes.parq'), engine='pyarrow')
 
 # Reprojected, cleaned and gzip (not snappy)
 
+df = pd.read_parquet(os.path.join(THIS_DIR, '..', 'earthquakes.parq'))
 
-# import numpy as np
-# import pandas as pd
-# from holoviews.util.transform import lon_lat_to_easting_northing
+cleaned_df = df.copy()
+cleaned_df['mag'] = df.mag.where(df.mag > 0)
+cleaned_df = cleaned_df.set_index("time")
+cleaned_df.index = pd.to_datetime(cleaned_df.index)
```

Comment:
Some notebooks set the time as index, some do not.

Will fix notebooks in a separate PR.